mdbook: 0.4.47 -> 0.4.48

trofi opened the pull request:
Changes: https://github.com/rust-lang/mdBook/blob/master/CHANGELOG.md#mdbook-0448


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

matthiasbeyer commented:
@NixOS/nixpkgs-merge-bot merge

Frostman commented:
@trofi could you please rebase to the master? The `cargoDeps = rustPlatform.fetchCargoVendor` section was replaced recently in https://github.com/NixOS/nixpkgs/commit/b6fb91950c34fe6eacd7c3afca74f7a638374d4f#diff-64d1380ec3303cb8cd9ce46eb90a204132598d5e0597529ab56fba4d63c264b5

Frostman reviewed (changes requested):
@trofi could you please rebase to the master? The `cargoDeps = rustPlatform.fetchCargoVendor` section was replaced recently in https://github.com/NixOS/nixpkgs/commit/b6fb91950c34fe6eacd7c3afca74f7a638374d4f#diff-64d1380ec3303cb8cd9ce46eb90a204132598d5e0597529ab56fba4d63c264b5

trofi committed:
mdbook: 0.4.47 -> 0.4.48

Changes: https://github.com/rust-lang/mdBook/blob/master/CHANGELOG.md#mdbook-0448

trofi commented:
> @trofi could you please rebase to the master? The `cargoDeps = rustPlatform.fetchCargoVendor` section was replaced recently in [b6fb919#diff-64d1380ec3303cb8cd9ce46eb90a204132598d5e0597529ab56fba4d63c264b5](https://github.com/NixOS/nixpkgs/commit/b6fb91950c34fe6eacd7c3afca74f7a638374d4f#diff-64d1380ec3303cb8cd9ce46eb90a204132598d5e0597529ab56fba4d63c264b5)

Rebased against current `master`. Thank you!

Frostman commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 395379 --package mdbook`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>mdbook</li>
  </ul>
</details>
